src/store.config.js

```diff
@@ -4,16 +4,61 @@ import reducers from './reducers/index';
 const storeConfigure = () => {
   const initialStore = {
     careers: [
-      { title: 'doctor',
-        tagline: 'saves lives',
-        description: 'this is the amazing doctor who is just really rich!',
-        image: 'doctor.svg'
+      { title: 'Doctor',
+        tagline: 'Promots and Improves Health',
+        description: 'A doctor is a professional who practises medicine, which is concerned with promoting, maintaining, or restoring health.',
+        image: 'doctor.svg',
```

Comment:
I'm guessing we don't have these images yet?

Reply:
Nope, we should decide on what type of images we want to include.